deployer/__main__.py

```diff
@@ -38,9 +57,10 @@ def deploy_support(cluster_name):
             cluster.deploy_support()
 
 
-def deploy_jupyterhub_grafana(cluster_name):
+def deploy_grafana_dashboards(cluster_name):
     """
-    Deploy grafana dashboards for operating a hub
+    Deploy grafana dashboards to a cluster that provide useful metrics
+    for operating a JupyterHub
```

Comment:
Can you clarify what dashboards are deployed? This is https://github.com/jupyterhub/grafana-dashboards right?